docs/developers/smart-contract-tutorial-hardhat.md

```diff
@@ -0,0 +1,177 @@
+# Deploying a Smart Contract to Celestia Ethermint with Hardhat
+<!-- markdownlint-disable MD013 -->
+
+In this guide you'll learn how to deploy a Solidity smart contract to Celestia Ethermint with [Hardhat](https://hardhat.org/).
+
+## About Hardhat
+
+Hardhat is a Solidity development environment built with Node.js that allows you to write tests and scripts using JavaScript.
+
+Hardhat gives you the tools necessary to deploy smart contracts, debug Solidity code, and run tests using a local test EVM network instead of having to deal with a live network.
+
+We'll use all Hardhat to create, test, and deploy our Solidity project.
+
+## Getting started
+
+To deploy to Celestia Ethermint you will need to request tokens from the test faucet.
+
+### Celestia Ethermint Faucet
+
+> USING THIS FAUCET DOES NOT ENTITLE YOU TO ANY AIRDROP OR OTHER DISTRIBUTION OF CELESTIA OR CELESTIA ETHERMINT TOKENS. MAINNET CELESTIA AND CELESTIA ETHERMINT TOKENS DO NOT CURRENTLY EXIST AND THERE ARE NO PUBLIC SALES OR OTHER PUBLIC DISTRIBUTIONS OF ANY MAINNET CELESTIA OR CELESTIA ETHERMINT TOKENS.
+
+You can request from Celestia Ethermint Faucet on the `#celestia-ethermint-faucet` channel on Celestia's Discord server with the following command:
+
+```text
+$request <EVM-WALLET-ADDRESS> 
+```
+
+### Initialize development environment
+
+First, create a new project in an empty directory :
+
+```sh
+npx hardhat init
+
+✔ What do you want to do? · Create a basic sample project
+✔ Hardhat project root: · <your-project-root>
+✔ Do you want to add a .gitignore? (Y/n) · y
+✔ Do you want to install this sample project's dependencies with npm (hardhat @nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers)? (Y/n) · y
+```
+
+Hardhat has created an example smart contract located at `contracts/Greeter.sol`.
+
+Next, update the configuration at `hardhat.config.js` to include the Celestia Ethermint network:
+
+```javascript
+require("@nomiclabs/hardhat-waffle");
+
+module.exports = {
+  solidity: "0.8.13",
+  networks: {
+    hardhat: {},
+    celestiaEthermint: {
+      url: "http://35.208.160.145:8545",
+      accounts: [process.env.PRIVATE_KEY],
+      chainId: 69420
+    }
+  }
+};
+```
+
+### Updating the contracts and tests
+
+Let's update the contracts to include a basic counter example. Create a new file in the `src` directory named `Counter.sol` and add the following code:
```

Comment:
Change `src` to `contracts` directory as that's what's available in Hardhat

Reply:
oh good catch!